ultraLatency.py

```diff
@@ -188,10 +187,28 @@
     sys.stdout.flush()
     startScanModule(ohboard, options.gtx, useUltra=True, debug=options.debug)
     printScanConfiguration(ohboard, options.gtx, useUltra=True, debug=options.debug)
+    amc13nL1A = (amc13board.read(amc13board.Board.T1, "STATUS.GENERAL.L1A_COUNT_HI") << 32) | (amc13board.read(amc13board.Board.T1, "STATUS.GENERAL.L1A_COUNT_LO"))
+    amcnL1A = getAMCL1Acount(amcboard)
+    ohnL1A = getLinkL1Acount(amcboard,options.gtx)
+    print "Initial L1A counts:"
+    print "AMC13: %s"%(amc13nL1A)
+    print "AMC: %s"%(amcnL1A)
+    print "OH%s: %s"%(link,ohnL1A)
     sys.stdout.flush()
-    scanData = getUltraScanResults(ohboard, options.gtx, LATENCY_MAX - LATENCY_MIN + 1, options.debug)
+    scanData = getUltraScanResults(ohboard, options.gtx, LATENCY_MAX - LATENCY_MIN + 1, options.debug, int(options.nevts), True)
```

Comment:
it's not a debug flag, it is isLatency flag which is defaulted to be False, but here in ultraLatency script has to be set to True